### DISCUSSION TITLE
Please help: Domain is in use by another Vercel account

### DISCUSSION THREAD

**Discussion by cayasso** [vercel/community#3488] [Help] [answered]
### Summary

Hi Vercel Team, I was testing your Platform Starter Kit at `https://app.vercel.pub` and I accidentally added my domain `ttager.com` under my [test account settings](https://app.vercel.pub/site/clku6y7do0001md08sukth0d8/settings/domains) then I tried to remove it do to the error above, but still got the error, there is no way/option to remove it from the team/account, I cleared the domain and save changes but I still get the error "domain is in use by another Vercel account"

Can you please help me removing the domain from `https://app.vercel.pub` team so I can properly add it to my vercel project?

Thank you!

<img width="1538" alt="image" src="https://github.com/vercel/community/assets/42518/bce24601-578c-425a-bba0-4f3107c42666">

Removed
<img width="1530" alt="image" src="https://github.com/vercel/community/assets/42518/058419f7-aab5-4910-b4d1-6b737a7bde63">

The error:
<img width="934" alt="image" src="https://github.com/vercel/community/assets/42518/1ecf4b86-0f8a-404e-b5c6-2ae43d85eb26">

### Steps to Reproduce

<img width="934" alt="image" src="https://github.com/vercel/community/assets/42518/1ecf4b86-0f8a-404e-b5c6-2ae43d85eb26">

**Comment by cayasso** [comment]
BTW the domain is already pointing to your nameservers, but the error above is preventing me to actually use it in my project.

**Comment by amyegan** [comment]
Hey @cayasso. Just follow the instructions to add a TXT record with your domain registrar to verify domain ownership. Then the domain will be associated with your account.

**Reply by cayasso** [reply]
Thanks @amyegan but I cannot add a DNS record because the domain is like in the limbo (in that other platform vercel account) atm, since I don't have access to it in Vercel even do it is pointing to ns1.vercel-dns.com, ns2.vercel-dns.com

**Reply by cayasso** [reply, marked as answer]
Problem solved, just did what you say, I had to change nameservers back to the original registrar and then add the TXT record for validation and then once verified I pointed back to vercel nameservers.
Thank you @amyegan

**Reply by mikhkosvn** [reply]
sorry, probably stupid question but you did add that on the side of your domain provider (lets say https://www.godaddy.com/), or on vercel https://vercel.com/account/domains/example.com?

**Reply by amyegan** [reply]
For TXT verification, you would need to add the record with your domain provider. Make sure you're also using their nameservers and have the appropriate A and CNAME records set.

I also want to make sure you're aware of our recent account structure update. Some people have run into a TXT verification notice after the update, and they solved it by moving their domain from their personal account to their new hobby team. You can find instructions for that and learn more about the account update in https://github.com/vercel/community/discussions/6298

**Reply by mikhkosvn** [reply]
thank you

**Comment by cayasso** [comment]
Actually this is not quite right yet, I just tested removing my domain from my vercel account and re-adding and I get the same error again, meaning that I have to go and do what I just did again. While this work for now, the permanent and proper solution would be to remove/detach my domain from `https://app.vercel.pub` vercel account. @amyegan can you pls remove the domain?

**Reply by amyegan** [reply]
I would still need to verify your ownership of the domain before I could make any changes. Please switch back to the original nameservers and add the TXT record so I know you have ownership of the domain 🙏

**Reply by cayasso** [reply]
@amyegan done I switched and added back the TXT record
```
_vercel vc-domain-verify=ttager.com,f23023e49175cfab9b4a
```

**Reply by amyegan** [reply]
Thanks for doing that. The domain has been removed from the other account so you can add it to your project.

**Reply by cayasso** [reply]
Awesome @amyegan thank you so much, have a great day!

**Reply by bdxoul** [reply]
my domain provider don't have option to use txt records,  I haven't used this account on any vercel account please help ,
![image](https://github.com/vercel/community/assets/12608621/0fd734ea-e19c-497d-8089-22ee06119e11)

My other domain from same register got linked easily

**Reply by bdxoul** [reply]
@amyegan
